### PR TITLE
fix(streaming_err): stop background repair task by force_terminate_repair API

### PR DESCRIPTION
**Pull request by amoskong**
After 'nodetool rebuild' is killed, the rebuild task will continue run in
background until complete. We can use force_terminate_repair API to stop the
background rebuild task. Then execute a new 'nodetool rebuild'.

Signed-off-by: Amos Kong <amos@scylladb.com>

Fixes: https://github.com/scylladb/scylla-cluster-tests/issues/2004

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -1781,6 +1781,14 @@ def is_streaming_started():
         if random.randint(0, 1) == 0:
             self.log.debug('Interrupt the task by pkill')
             self.target_node.remoter.run('sudo pkill -9 -f %s' % task, ignore_status=True)
+            self.log.debug("Stop background repair task by storage_service/force_terminate_repair API")
+            with DbEventsFilter(type='DATABASE_ERROR', line="repair's stream failed: streaming::stream_exception",
```
Comment (fruch) [inline sdcm/nemesis.py:1785]: we might start think, of turn this repair expected errors into a group to reuse, somehow.
cause this list is keep changing

**Comment by roydahan** [comment]
Don't we have this also in 4.0?

**Comment by fruch** [comment]
> Don't we have this also in 4.0?

those nemesis we merge few days ago.... and not yet merged to branch-4.0 (not sure they are planned to be merged or not)

File: sdcm/nemesis.py (hunk 1 above)
Comment (bentsi) [reply sdcm/nemesis.py:1785]: we need to put all grouped filters in one place, like on file. 
Created a task for that https://trello.com/c/V3Ca21uD/1893-group-common-events-filters

**Pull request merged**